### ISSUE TITLE
Message status page - Redesigned

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#930]
The message status page is problematic because it's not easy to understand.
https://notification.alpha.canada.ca/features/messages-status

- No French diagram
- Text not descriptive enough to complement diagram, text from diagram should be embedded in content (multi-modality)
- Colour contrast fails for delivered/failed
- There's an i <i ></i > tag in the alt text of the image that gets read out
- The pipe in the diagram is not clear what it's communicating
- Table content is not accessible, only reads the titles. Is there a better way to present this information?

https://docs.google.com/document/d/18W-nKAH-54wpNy5XSZeEA31my4XRMxpZRVJ3en4ThAA/edit

Latest wireframes on far right: https://www.figma.com/file/qSW5XL4PpjbC8SyZAl2bkS/Unauthenticated-pages?node-id=1%3A34

**Comment by anikbrazeau** [comment]
First draft in Figma here: https://www.figma.com/file/qSW5XL4PpjbC8SyZAl2bkS/Unauthenticated-pages?node-id=1%3A34

**Comment by ClementineHahn** [comment]
@brdunfield finished work, she's  waiting for an image to complete the card

**Comment by ClementineHahn** [comment]
@ioanacontu will send to @brdunfield today 22nd July
